src/state_point.cpp

```diff
@@ -1,16 +1,20 @@
 #include "openmc/state_point.h"
 
-#include <algorithm>
-#include <vector>
-
-#ifdef OPENMC_MPI
-#include "mpi.h"
-#endif
-
 #include "openmc/capi.h"
+#include "openmc/constants.h"
 #include "openmc/error.h"
+#include "openmc/hdf5_interface.h"
 #include "openmc/message_passing.h"
 #include "openmc/settings.h"
+#include "openmc/simulation.h"
+#include "openmc/tallies/tally.h"
+
+#include "xtensor/xbuilder.hpp" // for empty_like
+#include "xtensor/xview.hpp"
+
+#include <algorithm>
+#include <string>
+#include <vector>
```

Comment:
This `#include` order is reversed from our conventional standard library > third-party > openmc-specific ordering.  Do you want to move to this convention throughout the C++ code?  If so, it might be better to do that in a separate PR so we don't have two conventions coexisting.

Reply:
Yeah, that is my preferred convention now, but I can switch this back and handle all of them in a separate PR.